NEWS

```diff
@@ -6,6 +6,15 @@ Please send bug reports, questions and suggestions to
 Version 4.4.3
 * Fix the value of SUNMD5_MAX_ROUNDS.
 * Add generated C++-guards to <xcrypt.h>.
+* Add --enable-obsolete-api-enosys configure option.
+  If enabled, this option replaces the obsolete APIs (fcrypt,
+  encrypt{,_r}, and setkey{,_r}) with POSIX compliant stubs setting
+  "errno" to "ENOSYS", instead of actually performing any real operation.
```

Comment:
"... with stubs that set errno to ENOSYS and return without performing any real operations. This allows one to disable DES support while preserving POSIX compliance."

Reply:
Fixed.